auction_seller_invoice/views/auction_item_invoice_views.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- pylint:disable=deprecated-data-xml-node -->
+<odoo>
+    <record id="account_move_seller_form_view" model="ir.ui.view">
+        <field name="name">account.move.form</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form" />
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <xpath expr="//page[@id='other_tab']" position="replace" />
+            <xpath
+                expr="//group[@id='header_right_group']/field[@name='date']"
+                position="replace"
+            />
+            <xpath expr="//label[@for='invoice_payment_term_id']" position="replace" />
+            <xpath
+                expr="//field[@name='invoice_payment_term_id']"
+                position="attributes"
+            >
```

Comment:
```suggestion
            <-- Cannot use 'replace' here since the value is used elsewhere in the view. -->
            <xpath
                expr="//field[@name='invoice_payment_term_id']"
                position="attributes"
            >
```